homeassistant/components/zwave_js/device_action.py

```diff
@@ -165,7 +166,7 @@ async def async_get_actions(hass: HomeAssistant, device_id: str) -> list[dict]:
                 CONF_TYPE: SERVICE_SET_CONFIG_PARAMETER,
                 ATTR_CONFIG_PARAMETER: config_value.property_,
                 ATTR_CONFIG_PARAMETER_BITMASK: config_value.property_key,
-                CONF_SUBTYPE: f"{config_value.value_id} ({config_value.property_name})",
+                CONF_SUBTYPE: generate_config_parameter_subtype(config_value),
```

Comment:
This is not a breaking change since we don't use this value for anything in the service and the value can be any string, right?

Reply:
I don't believe it is. Existing device automations will keep their old strings but any new ones will have the new format

Reply:
Yeah and changing this string doesn't matter as it's not used for any logic.